packages/terra-consumer-nav/src/components/nav-logo/NavLogo.jsx

```diff
@@ -43,10 +43,17 @@ const NavLogo = ({
 }) => {
   const imgProps = {
     className: cx('img'),
-    ...url && { src: url },
+    ...url ? {
+      src: url,
+      alt: altText,
+    } : {
+      children: altText,
```

Comment:
where is this children rendered?

Reply:
i see, it is rendered inside p tag

Reply:
this is rendered inside the 'p' tag generated inside `LogoElement` when a url is not passed